contracts/admin/build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    if option_env!("LOCAL_NET_ADMIN_CONTRACT")
+        .map_or(false, |value| ["1", "y", "Y"].contains(&value))
+    {
+        println!("cargo:rustc-cfg=feature=\"admin_contract_exec\"");
```

Comment:
two questions:
- why do we go through an env variable to enable a feature?
- I doubt we need such a shortcut of having an `execute` to `sudo` bridge. More code and potential human error are among the cons of it. We have mastered in gov proposals so would ask of removing this feature.

Reply:
This was meant to simplify the process for local testing, removing the need to run migrations through government proposals.

Removed as you have asked.